sklearn/linear_model/tests/test_perceptron.py

```diff
@@ -67,3 +67,13 @@ def test_undefined_methods():
     clf = Perceptron(max_iter=100)
     for meth in ("predict_proba", "predict_log_proba"):
         assert_raises(AttributeError, lambda x: getattr(clf, x), meth)
+
+
+def test_perceptron_l1_ratio():
```

Comment:
can we get a better test?

Also, I feel like the perceptron documentation could probably be improved. Our perceptron class doesn't really implement only a perceptron. Has anyone else implemented something like this?

Reply:
Not that I'm an example to follow on this, but let's try to be more descriptive in our comments ;) What kind of test would you like to see @amueller?